examples/static/source/Root.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Link } from 'react-router'
 
 const addBase = (path) => (
-  `${(process.env.BASE_PATH || '').replace(/\/$/, '') || '/'}${path}`
+  `${(process.env.BASE_PATH || '').replace(/\/$/, '') || '/'}${path.slice(1)}`
```

Comment:
I'm actually killing this in an upcoming PR which will version bump boiler-room runner :)